test/inductor/test_aot_inductor.py

```diff
@@ -58,18 +59,20 @@ class AOTInductorModelRunner:
     @classmethod
     def compile(cls, model, example_inputs, options=None, constraints=None):
         # The exact API is subject to change
-        so_path, exported = torch._export.aot_compile(
+        so_path = torch._export.aot_compile(
             model,
             example_inputs,
             options=options,
             constraints=constraints,
             remove_runtime_assertions=True,
         )
-        return so_path, exported
+        return so_path
 
     @classmethod
     def load(cls, so_path, example_inputs):
-        is_cpu = all(x.device.type == "cpu" for x in example_inputs)
+        is_cpu = all(
+            x.device.type == "cpu" for x in pytree.tree_flatten(example_inputs)[0]
```

Comment:
Are you only checking the first input now?

Reply:
pytree.tree_flatten returns a flattened list of example_inputs and the tree_spec for how it flattened the original list. So indexing [0] just gets the entire flattened list.